public/less/themes/arc-green.less

```diff
@@ -85,6 +85,59 @@ body {
     color: #9e9e9e;
 }
 
+/* firefox scroll bars */
+
+html,
+body,
+* {
```

Comment:
`html` and `body` should not be necessary, they are matched by `*`.

Reply:
you right,
done